### DISCUSSION TITLE
Student/Project Management Questions

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#433] [Q&A]
@pennylenger this is a good place to ask questions as they come up!

**Comment by CECSpecialistI** [comment]
@cspayne this is a good place to ask questions as they come up!

**Comment by cspayne** [comment]
@CECSpecialistI 
I'm looking at [584 - Accumulation and Frequency of Use Note](https://docs.google.com/spreadsheets/d/1cqc-HkrnQYVyuFnjd7AcmurK-j9U5wp2gvp__nK9y8k/edit#gid=1611682411) right now, and am wondering if there are any clues/techniques for determining whether a field or subfield is referring to a work, expression, manifestation, or item. 

For this field specifically (and I can post this on the 584 issue if that is a better place for it), [the MARC page](https://www.loc.gov/marc/bibliographic/bd584.html) makes it seem like this field is used for collections/parts of collections, so could this be describing the work (where more information/content is being added to the work) even though the accumulation is measured in volume/time period?

**Reply by GordonDunsire** [reply]
I agree that 584 seems to be applicable to collections only.

More accurately, 584 is applicable to a collection Work:

- $a Accumulation: maps to rdaw:P10368 "has frequency"
- $b Frequency of use: is way out of scope for RDA and nearly every cataloguing code I know of
- $3 Materials specified: maps to rdaw:P10088 "has title of work"
- $5 Institution to which field applies: maps to rdaw:P10631 "has collector corporate body"
- $a + $b + $3 + $5: maps to rdaw:P10330 "has note on work"

$3 maps to a general title because the examples are of generic titles that would normally be augmented with the name of the holding institution. This can be made more specific by using $5:
$5 + $3 maps to rdaw:P10223 "has preferred title of work".

The note concatenates the subfields in the order in which they appear; punctuation is added and replaced as required.

Example:

collectionWork1 rdaw:P10088 "General subject files" ;
   rdaw:P10223 "Institution A General subject files";
   rdaw:P10368 "45 cu. ft. average annual accumulation 1970-1979.";
   rdaw:P10368 "5.4 cu. ft. average monthly accumulation, 1979-82.";
   rdaw:P10368 "Current average month accumulation is 2 cu. ft.";
   rdaw:P10631 "Institution A";
   rdaw:P10330 "General subject files: 45 cu. ft. average annual accumulation 1970-1979;  5.4 cu. ft. average monthly accumulation, 1979-82; Current average month accumulation is 2 cu. ft." .

Other information about the collection work is taken from the rest of the MARC 21 record which needs to separate out the collection work from the collection manifestation. In other words, I think the existence of 584 in a record indicates that a special map for collections should be used in the transform.

**Reply by cspayne** [reply]
Thank you! This was helpful. I have a few related questions, but I will move over to [the issue](https://github.com/uwlib-cams/MARC2RDA/issues/235), so it can easily be tracked.

**Comment by sarahruska** [comment]
@CECSpecialistI, @cspayne - I have a clarifying question on the Decisions Index [I.A.1. "Delete" status](https://github.com/uwlib-cams/MARC2RDA/wiki/Decisions-Index#ia1-delete-status). Should justifications for recording both "Delete" _and_ "Not Mapped" status go in "Notes--Uncategorized"? The description with "Reasons for not mapping..." had me questioning if this is supposed to apply to "Not Mapped" as well.

**Reply by CECSpecialistI** [reply]
That's a good question. We haven't specified before. Yes?

**Reply by sarahruska** [reply]
Yeah, from what I can tell we haven't specified for the "Not Mapped" status before

**Reply by CECSpecialistI** [reply]
Did we ever answer this question? I think we should.

**Reply by sarahruska** [reply]
Thanks for catching this - hadn't closed it out! I added this clarification to the Decisions Index
